### ISSUE TITLE
[Feature Request] Import SVG gradient fills

### ISSUE THREAD

**Issue by michacassola** [MaurycyLiebner/enve#223]
Please make it possible to import SVGs with gradient fills.

**Comment by AlexKiryanov** [comment]
Good day! This function is there. Here. Take a look.
![image](https://user-images.githubusercontent.com/65244629/111129041-ac6f0180-8586-11eb-83eb-24970c21085c.png)
![image](https://user-images.githubusercontent.com/65244629/111129163-ce688400-8586-11eb-8464-ab66050208c5.png)
The problem is with the gradient blending modes. They don't work yes.

**Comment by michacassola** [comment]
Hey @AlexKiryanov, thanks again for this awesome 2D animation tool! I am very excited to have found it and the more I check out your videos I fall in love with it.

Unfortunately animating color fills, especially gradient fills with their points and point locations is one of the most important things for me. To illustrate here a GIF that I made with Animatron. How could I do that with enve?
(Little side question, I have 0 french knowledge, how do you properly pronounce enve)?

![Foundyn-Animation-2 1](https://user-images.githubusercontent.com/19753510/111786738-6b485b80-88be-11eb-852a-2943569ef657.gif)

**Comment by AlexKiryanov** [comment]
Hey! Yes thank @MaurycyLiebner for this awesome 2D animation tool!)) 
I can offer you a raster effect for creating a gradient with 4 colors, in which you can assign the colors you want and control their position.
![image](https://user-images.githubusercontent.com/65244629/111798375-1109c380-88db-11eb-9abf-a66766fd2726.png)

**Comment by AlexKiryanov** [comment]
Or a radial gradient with a choice of colors and their position.
![image](https://user-images.githubusercontent.com/65244629/111803475-1f0e1300-88e0-11eb-9e6b-1e03c1dfe0f4.png)

**Comment by michacassola** [comment]
Seems like a good enough workaround for now. How do you do it you enve pro. :)
Would you please provide better import and control natively through SVG import and editing please @MaurycyLiebner ?

**Comment by michacassola** [comment]
Nope, an object inside another one is not unspported as it seems, using Inkscape it did work, so it must be in the SVG spec. Please support it, will make a seperate issue.

**Comment by AlexKiryanov** [comment]
As I already wrote, this effect is only suitable for raster format. it will not be in the exported svg.

**Comment by michacassola** [comment]
After some investigation there seems to be a workaround to import fills:
- Export the SVG with linear gradient fill
- Import in enve
- Select the object, that is without fill
- Add a gradient fill in the enve right hand fill window, where you will also find your predefined gradient that for some reason was imported but not set as fill of the object it was the fill of
- There you can make it radial again if you'd like

**Comment by AlexKiryanov** [comment]
@michacassola  
Good evening! To be honest, I don't understand what your problem is. I have gradients imported without any problems. The only thing I am missing in them is changing the position of the color in the gradient. 
![image](https://user-images.githubusercontent.com/65244629/111882590-70450200-89c7-11eb-8652-f4092a9d8031.png)
![image](https://user-images.githubusercontent.com/65244629/111882639-a7b3ae80-89c7-11eb-908c-be859bc04ae2.png)
I have attached the svg file
[drawing2.zip](https://github.com/MaurycyLiebner/enve/files/6176224/drawing2.zip)
Therefore, it will be difficult to make exactly the same gradient animation that you sent in the gif. Certainly possible, but difficult. 
You can animate the gradient but only the extreme points.

Maybe send your file for an example, so that I understand better ...? I'm just afraid I can still understand correctly because of the language barrier.

**Comment by michacassola** [comment]
I use Affinity Designer to make my SVGs. Inkscape is not an option for me. I work on Windows 10.
Here a sample file: [Foundyn-Logo-1.3-combined-curve-radial-gradient-1.svg.zip](https://github.com/MaurycyLiebner/enve/files/6177861/Foundyn-Logo-1.3-combined-curve-radial-gradient-1.svg.zip)

**Comment by AlexKiryanov** [comment]
Now I understand what the problem is)
